Source/Editor/Foundation/SceneViewTab.cpp

```diff
@@ -511,6 +516,37 @@ void SceneViewTab::FocusSelection(SceneSelection& selection)
             OnLookAt(this, *page, activeNode->GetWorldPosition());
     }
 }
+void SceneViewTab::CreatePrefabFile(SceneSelection& selection)
+{
+    if (Node* activeNode = selection.GetActiveNode())
+    {
+        if (SceneViewPage* page = GetPage(activeNode->GetScene()))
+        {
+            if (activeNode)
+            {
+                eastl::string prefabFileName = activeNode->GetName();
+
+                if (prefabFileName.length() < 1)
+                    prefabFileName = prefabFileName.sprintf("%s %d", "PrefabNodeID_", activeNode->GetID());
```

Comment:
Use `Format()` function for string formatting. `Format("PrefabNodeID_{}", activeNode->GetID())`.